include/ensmallen_bits/nsga2/nsga2.hpp

```diff
@@ -168,9 +169,13 @@ class NSGA2 {
   //! Modify value of upperBound.
   arma::vec& UpperBound() { return upperBound; }
 
-  //! Retrieve the best front (the Pareto frontier).  This returns an empty vector until `Optimize()`
-  //! has been called.
-  const std::vector<arma::mat>& Front() const { return bestFront; }
+  //! Retreive the pareto optimal points in variable space. This returns an empty cube
```

Comment:
```suggestion
  //! Retrieve the Pareto optimal points in variable space. This returns an empty cube
```

Minor spelling fix and captial Pareto just to be consistent.